tests/ctst/steps/utils/kubernetes.ts

```diff
@@ -445,3 +460,59 @@ export async function createSecret(
         throw err;
     }
 }
+
+
+export async function getMongoDBConfig(
+    world: Zenko,
+    namespace = 'default',
+) : Promise<{
+    replicaSetHosts: string[] | string;
+    username: string;
+    password: string;
+    database: string;
+}> {
+    // Get the MongoDB configuration from the secret
+    // Equivalent of doing :
+    // MONGODB_REPLICASET=$(kubectl get secrets -l app.kubernetes.io/name=connector-cloudserver-config 
+    //  -o jsonpath='{.items[0].data.config\.json}' | base64 -di | jq -r .mongodb.replicaSetHosts)
+
+    const coreClient = createKubeCoreClient(world);
+    try {
+        // Get secrets with the specific label
+        const secretsList = await coreClient.listNamespacedSecret(
+            namespace,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            'app.kubernetes.io/name=connector-cloudserver-config'
+        );
+        if (!secretsList.body.items || secretsList.body.items.length === 0) {
+            throw new Error('No secrets found with label app.kubernetes.io/name=connector-cloudserver-config');
+        }
+
+        const secret = secretsList.body.items[0];
+        const configJsonBase64 = secret.data?.['config.json'];
+        
+        if (!configJsonBase64) {
+            throw new Error('config.json not found in secret data');
+        }
+
+        // Decode base64 and parse JSON
+        const configJson = Buffer.from(configJsonBase64, 'base64').toString('utf-8');
+        const config = JSON.parse(configJson);
+
+        // Extract MongoDB configuration values
+        const mongodbConfig = {
+            replicaSetHosts: config.mongodb?.replicaSetHosts,
+            username: config.mongodb?.authCredentials?.username,
+            password: config.mongodb?.authCredentials?.password,
+            database: config.mongodb?.database,
+        };
+
+        return mongodbConfig;
+    } catch (err) {
+        world.logger.error('Error getting MongoDB config from secret', {namespace, err});
```

Comment:
can be a debug log: if we throw an error, the test will auto dump all logs in this case, if it failed.